Netflix/Screens/Onboarding/OnboardingViewController.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  OnboardingViewController.swift
+//  Netflix
+//
+//  Created by Tanya Samastroyenka on 25.05.2022.
+//
+
+import UIKit
+import SnapKit
+
+class OnboardingViewController: UIViewController, UIPageViewControllerDataSource, UIPageViewControllerDelegate {
+    
+    var pageController: UIPageViewController!
+    var controllers = [UIViewController]()
+
+    private let backgroundImageView = UIImageView(image: UIImage(named: Asset.Assets.onboarding.name))
+    
+    private let signInButton: UIButton = {
+        let button = UIButton()
+        button.setTitle("SIGN IN", for: .normal)
+        button.backgroundColor = Asset.Colors.onboardingButtons.color
+        button.setTitleColor(.white, for: .normal)
+        return button
+    }()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        view.addSubview(backgroundImageView)
+        backgroundImageView.alpha = 0.4
+        view.addSubview(signInButton)
+        pageController = UIPageViewController(transitionStyle: .scroll, navigationOrientation: .horizontal, options: nil)
+        pageController.dataSource = self
+        pageController.delegate = self
+
+        addChild(pageController)
+        view.addSubview(pageController.view)
+
+        let firstPageViewController = FirstPageViewController()
+        let secondPageViewController = SecondPageViewController()
+
+        controllers.append(firstPageViewController)
+        controllers.append(secondPageViewController)
+
+        pageController.setViewControllers([controllers[0]], direction: .forward, animated: false)
```

Comment:
Please avoid using [0] - you can `get index out of range` exception
```suggestion
        if let firstViewController = controllers.first {
                pageController.setViewControllers([firstViewController], direction: .forward, animated: false)
        }
```
Or
`Array(t.prefix(1)` to get first element as an array